### DISCUSSION TITLE
Optimism new version sync frozen.

### DISCUSSION THREAD

**Discussion by BaidocVlad** [ethereum-optimism/developers#520] [Node Operators]
### Are you running the most up to date node software?

- [X] Yes

### Did you check the documentation?

- [X] Yes

### Did you check for duplicate questions?

- [X] Yes

### Issue Description

I started a new RPC node with the latest version but the sync in frozen at the same block. I deleted all the data an start it again and the problem appear again.

### Protocol Description

Geth
Version: 1.101315.2-stable
Git Commit: 7c2819836018bfe0ca07c4e4955754834ffad4e0
Git Commit Date: 20240604
Upstream Version: 1.13.15-stable
Architecture: amd64
Go Version: go1.21.11
Operating System: linux
GOPATH=
GOROOT=

/////////////////

OP-NODE version: [v1.7.7](https://github.com/ethereum-optimism/optimism/tree/v1.7.7)

### Node Logs

Jul 09 08:10:07 moonlet-rpc-mainnet-04 geth[2362447]: DEBUG[07-09|08:10:07.259] Ancient blocks frozen already            number=117,387,667 hash=ce4dba..9399e8 frozen=117,297,668
Jul 09 08:11:07 moonlet-rpc-mainnet-04 geth[2362447]: DEBUG[07-09|08:11:07.260] Ancient blocks frozen already            number=117,387,667 hash=ce4dba..9399e8 frozen=117,297,668
Jul 09 08:12:07 moonlet-rpc-mainnet-04 geth[2362447]: DEBUG[07-09|08:12:07.260] Ancient blocks frozen already            number=117,387,667 hash=ce4dba..9399e8 frozen=117,297,668
Jul 09 08:13:07 moonlet-rpc-mainnet-04 geth[2362447]: DEBUG[07-09|08:13:07.261] Ancient blocks frozen already            number=117,387,667 hash=ce4dba..9399e8 frozen=117,297,668
Jul 09 08:14:07 moonlet-rpc-mainnet-04 geth[2362447]: DEBUG[07-09|08:14:07.262] Ancient blocks frozen already            number=117,387,667 hash=ce4dba..9399e8 frozen=117,297,668
Jul 09 08:15:07 moonlet-rpc-mainnet-04 geth[2362447]: DEBUG[07-09|08:15:07.263] Ancient blocks frozen already            number=117,387,667 hash=ce4dba..9399e8 frozen=117,297,668
Jul 09 08:16:07 moonlet-rpc-mainnet-04 geth[2362447]: DEBUG[07-09|08:16:07.264] Ancient blocks frozen already            number=117,387,667 hash=ce4dba..9399e8 frozen=117,297,668
Jul 09 08:17:07 moonlet-rpc-mainnet-04 geth[2362447]: DEBUG[07-09|08:17:07.264] Ancient blocks frozen already            number=117,387,667 hash=ce4dba..9399e8 frozen=117,297,668
Jul 09 08:18:07 moonlet-rpc-mainnet-04 geth[2362447]: DEBUG[07-09|08:18:07.265] Ancient blocks frozen already            number=117,387,667 hash=ce4dba..9399e8 frozen=117,297,668
Jul 09 08:19:07 moonlet-rpc-mainnet-04 geth[2362447]: DEBUG[07-09|08:19:07.266] Ancient blocks frozen already            number=117,387,667 hash=ce4dba..9399e8 frozen=117,297,668

**Comment by opfocus** [comment]
Can you provide the startup commands for your op-geth and op-node?

**Reply by BaidocVlad** [reply]
op-node

ExecStart=/usr/local/bin/op-node \
  --rpc.addr=0.0.0.0 \
  --l1="OUR NODE"  \
  --l1.rpckind=any \
  --l1.beacon="OUR NODE" \
  --l2=http://127.0.0.1:8751 \
  --p2p.listen.tcp=9998 \
  --l2.jwt-secret=./jwt.txt \
  --network=op-mainnet \
  --rpc.port=8845 \
  --l2.jwt-secret=/home/optimism/optimism/jwt.txt \

///////////////////////////////////////

op-geth

ExecStart=/usr/local/bin/geth \
  --ws \
  --ws.port=9992 \
  --ws.addr=0.0.0.0 \
  --ws.origins="*" \
  --ws.api=eth,net,debug \
  --http \
  --op-network=op-mainnet \
  --http.port=8549 \
  --rollup.halt=major \
  --http.addr=0.0.0.0 \
  --http.vhosts="*" \
  --http.corsdomain="*" \
  --http.api=eth,net,debug \
  --authrpc.addr=localhost \
  --authrpc.jwtsecret=/home/optimism/optimism/jwt.txt \
  --authrpc.port=8751 \
  --authrpc.vhosts="*" \
  --verbosity=3 \
  --datadir=/home/optimism/geth-data \
  --rollup.disabletxpoolgossip=true \
  --rollup.sequencerhttp=https://mainnet-sequencer.optimism.io/ \
  --nodiscover \
  --syncmode=snap \
  --maxpeers=0 \
  --verbosity=4 \
  --metrics \
  --metrics.addr=0.0.0.0 \
  --metrics.port=6770 \
  --networkid=10 \
  --pprof \
  --pprof.addr=0.0.0.0 \
  --pprof.port=6760 \
  --gcmode=archive \
  --nat=none \
  --snapshot=false

**Comment by joohhnnn** [comment]
When your op-geth opts for snap synchronization, this is normal behavior. For more details, you can refer to [this discussion on GitHub](https://github.com/ethereum/go-ethereum/issues/20286#issuecomment-553745857). You don't need to delete any data; just give it some more time to let geth complete the snap synchronization. Afterwards, check the sync status of geth. Restarting is also an option as mentioned in [another GitHub comment](https://github.com/ethereum/go-ethereum/issues/20424#issuecomment-562386364).
